### ISSUE TITLE
Add man pages

### ISSUE THREAD

**Issue by Polpetta** [Polpetta/jkk#14]
While I know we have a lot of implementation still to do, we also need to check out for user documentation. In UNIX world, everyone expects a CLI to have at least a man page explaining what it does.

Here we need to gradually add it to the current project!

**Comment by Polpetta** [comment]
Update: pandoc can generate man pages from Markdown documentation, thus we can take markdown-flavoured documentation and convert it in man pages.
Example:
```
man -l - <<< $(pandoc --to=man Home.md)
```

In the next days I'll create a `/docs` folder in the project, and then setup a task in Gradle to run Pandoc and generate a `build/man` folder with all the documentation inside. Finally, I'll move the WIki section in the repository project, so we can check it with pull requests & stuff

**Comment by apontini** [comment]
I think it’s correct to explicit all this things now, before we start to develop new commands. This way we’ll have a clean path ahead, without backtracking to implement extra features

**Comment by Polpetta** [comment]
Yep! On top of that, once we define the commands we can work on them separately without interference
